src/core/selfcare_be.tf

```diff
@@ -0,0 +1,153 @@
+module "selfcare_be" {
+  source = "git::https://github.com/pagopa/azurerm.git//app_service?ref=v2.0.2"
+
+  name = "${var.location}-${var.env_short}"
+
+  ftps_state = "AllAllowed"
+
+  plan_name     = "${var.location}-plan"
+  plan_sku_size = var.app_service_plan_sku
+  plan_sku_tier = var.app_service_plan_sku_tier
+  plan_kind     = var.app_service_plan_kind
```

Comment:
better to use specific var names, for example

devportal_plan_sku
devportal_plan_sku_tier
devportal_plan_kind